<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FUstZapr</class>
 <widget class="QWidget" name="FUstZapr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование установок запроса</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/setting_tools.png</normaloff>:/res/1/setting_tools.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Название запроса:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_naz"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Дата и время создания :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dt_dat">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Название потребителя:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_nazpotr">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbtn_potr">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/1/bullet_edit.png</normaloff>:/res/1/bullet_edit.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Начальная дата для включения информации:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dt_dtn"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Конечная дата для включения информации:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dt_dtk"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Дата последнего обновления запроса:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dt_dtt">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="keyboardTracking">
                 <bool>true</bool>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Тип конечной даты</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="rb_ust">
               <property name="text">
                <string>Установленная</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb_tek">
               <property name="text">
                <string>Текущая</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb_teksut">
               <property name="text">
                <string>Текущая - сутки</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Количество суток:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_kolsut"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_pravt">
        <property name="text">
         <string>Признак автоматического обновления запроса</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="cb_ds">
            <property name="text">
             <string>Выдавать измерительную информацию ДС</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_kos">
            <property name="text">
             <string>Выдавать измерительную информацию КОС</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_dzs">
            <property name="text">
             <string>Выдавать измерительную информацию ДЗС</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_meteo">
            <property name="text">
             <string>Выдавать измерительную информацию по метеоданным</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_bds_l">
            <property name="text">
             <string>Выдавать  информацию БДС (необработанные файлы)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_ppsi">
            <property name="text">
             <string>Выдавать исходную информацию с ППСИ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="cb_bds_o">
            <property name="text">
             <string>Выдавать  информацию БДС (файл измерений)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_bds_n">
            <property name="text">
             <string>Выдавать  информацию БДС (навигационные файлы)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_kvi">
            <property name="text">
             <string>Выдавать  информацию ДДС (КВИ)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_kvno">
            <property name="text">
             <string>Выдавать  информацию ДДС (КВНО)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCheckBox" name="cb_alti">
              <property name="text">
               <string>Выдавать  информацию ВРВ (ALTI)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_pralti">
              <property name="text">
               <string>Переименовать для UGN0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_cal">
            <property name="text">
             <string>Выдавать  информацию ВРВ (CAL1,CAL2)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QCheckBox" name="cb_sp3">
          <property name="text">
           <string>Выдавать эфемеридно-временную информацию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_sol">
          <property name="text">
           <string>Выдавать протоколы оценки качества ИИ ВРВ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>73</height>
           </size>
          </property>
          <property name="title">
           <string>Версия RINEX файла для БДС</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="rb_3">
             <property name="text">
              <string>3.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_2">
             <property name="text">
              <string>2.11</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Путь к директории для выдаваемой информации:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_path">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_path">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/folder.png</normaloff>:/res/1/folder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/OK.png</normaloff>:/res/OK.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Cancel">
          <property name="text">
           <string>Отменить</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/No-entry.png</normaloff>:/res/No-entry.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Exit">
          <property name="text">
           <string>Выход</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>45</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
